src/IntegrationUtils.ts

```diff
@@ -0,0 +1,129 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+import * as vscode from 'vscode';
+import * as extension from './extension';
+import * as utils from './CamelKJSONUtils';
+import * as path from 'path';
+import * as child_process from 'child_process';
+import { getConfigMaps, getSecrets } from './ConfigMapAndSecrets';
+import * as k8s from 'vscode-kubernetes-tools-api';
+
+export async function startIntegration(context: vscode.Uri): Promise<any> {
+	return new Promise <any> ( async (resolve, reject) => {
+		await vscode.window.showQuickPick([
+			'Basic - Apache Camel K Integration (no ConfigMap or Secret)', 
```

Comment:
I would opt for creating a constant for these 3 values because they are later on used in a switch too. Given that the values we have here are not short simple words the duplication might easily break the switch one the quick pick values are modified. What do you think?